warning: non-void function does not return a value

Assuming one wants to compile utest.h cleanly without warnings, here's a warning report I've noticed:

```
❯ bazel test ...
INFO: From Compiling external/sheredom_utest/decoy_main.c:
In file included from bazel-out/freebsd-fastbuild/bin/external/sheredom_utest/decoy_main.c:1:
external/sheredom_utest/utest.h:266:1: warning: non-void function does not return a value [-Wreturn-type]
}
^
1 warning generated.
```

Sysinfo:
```
❯ clang --version
clang version 14.0.0
Target: x86_64-portbld-freebsd13.0
Thread model: posix
InstalledDir: /usr/local/llvm-devel/bin
```

--- a/utest.h
+++ b/utest.h
@@ -44,6 +44,7 @@
 */
 #pragma warning(disable : 4711)
 
+#if _MSC_VER > 1900
 /*
   Disable warning about preprocessor macros not being defined in MSVC headers.
 */
@@ -58,6 +59,7 @@
   Disable warning about pointer or reference to potentially throwing function.
 */
 #pragma warning(disable : 5039)
+#endif
 
 #pragma warning(push, 1)
 #endif

